browser/js/components/GraderAssessments.js

```diff
@@ -1,12 +1,15 @@
 'use strict'
 
 import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux'
 import Paper from 'material-ui/Paper';
 import RaisedButton from 'material-ui/RaisedButton';
 import FlatButton from 'material-ui/FlatButton';
 import FontIcon from 'material-ui/FontIcon';
+import Chip from 'material-ui/Chip'
 import { Card, CardActions, CardHeader, CardText } from 'material-ui/Card';
 import { RadioButton, RadioButtonGroup } from 'material-ui/RadioButton';
+import { getUserAssessments } from '../actions/userAssessmentActions'
 
 const styles = {
```

Comment:
Consider moving this to another file. Reason one: this current file is getting a little long. Reason two: these styles are likely to just keep growing. Reason three: it seems like you might want to be able to work on the styles and the component logic both independently and concurrently, so having them in different files can help you avoid future unnecessary merge conflicts.